CrisisTextLine/Elmer
# added ability to open one-time shifts

#### What's this PR do?
Just as weekly shifts are opened in `RecurOpenShifts.js`, this script automates opening one-time shifts.

Also, the config with the max number of open shifts was changed to a percentage that is calculated against the total number of CC's on When I Work to make it scale automatically so that we don't necessarily have to manually change the config as the number of active CC's grows. 

#### Where should the reviewer start?
`elmer/jobs/scheduling/RecurOpenShifts.js`
#### How should this be manually tested?
add `global.KEYS = require('../../keys.js');` and `global.CONFIG = require('../../config.js');` to the top of `RecurOpenShifts.js`, run locally with `node RecurOpenShifts` and console.log out anything you want. Check When I Work for shifts opening today, a week from now, and 2 weeks from now. 

The script does not work on odd hours, so comment out the `now` variable

#### Any background context you want to provide?
If you do test this, Heather has given us free reign to open shifts, so have at it.
#### What are the relevant tickets?
https://admin.crisistextline.org/jira/browse/AT-116
#### Questions:
Do you think we need to have percentages instead of hard numbers for the config? Are we changing it that much?
